client/js/admin.js

```diff
@@ -169,13 +169,14 @@ const attachHandles = function() {
 	$(".copy-path").click(function(e) {
 		e.stopPropagation()
 		const path = $(this).attr("path")
-		const cb = $("#copy-buffer")
-		cb.text(path)
-		const range = document.createRange()
-		range.selectNode(cb[0])
-		window.getSelection().empty ? window.getSelection().empty() : undefined
-		window.getSelection().removeAllRanges ? window.getSelection().removeAllRanges() : undefined
-		window.getSelection().addRange(range)
+		const $copyBuffer = $("#copy-buffer")
```

Comment:
I don't like the use of $ to prefix the variable name. What is the rationale? Is it just to indicate that it's a jquery object?